backend/src/main/java/ca/mcgill/purposeful/model/CollaborationRequest.java

```diff
@@ -0,0 +1,117 @@
+package ca.mcgill.purposeful.model;
+
+import ca.mcgill.purposeful.model.VerificationRequest.Status;
+import jakarta.persistence.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+/**
+ * The CollaborationRequest class, the model for regular user's collaboration requests to an idea in
+ * the database
+ */
+@Entity
+public class CollaborationRequest {
+
+  // ------------------------
+  // CollaborationRequest Attributes
+  // ------------------------
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  @GenericGenerator(name = "uuid", strategy = "uuid2")
+  private String id;
+
+  @Column(nullable = false)
+  private Status status;
+
+  @Column(nullable = true)
```

Comment:
I think we should leave the additionalContact information or the message non nullable, so that there is always some contents to display in the collaboration request. 

If we wanted to from the business logic perspective, we could have default values, eg. email and a generic message, so the user could have a choice to just send a request. But I think its good to enforce the presence of content from the backend